docs/integrations/webhooks/create-a-webhook.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Create a Clerk webhook in your Next.js application
+description: Learn how to create a Clerk webhook with Svix.
+---
+
+# Create a Clerk webhook in your Next.js application
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk + Next.js application",
+      link: "https://clerk.com/docs/quickstarts/nextjs"
+    }
+  ]}
+>
+- Listen for an event in the Clerk Dashboard
+- Create a webhook handler that
+  - uses Svix to verify the webhook signature
+  - receives the webhook's payload
+- Test the webhook locally using ngrok and the Clerk Dashboard
+- Add logic to your application to trigger the webhook
+</TutorialHero>
+
+In this guide, you will learn how to create a Clerk webhook in your Next.js application. You will listen for the `user.updated` event by creating a webhook endpoint in the Clerk Dashboard, creating a webhook handler in your Next.js application, and testing the webhook locally using ngrok and the Clerk Dashboard.
+
+This guide can be adapted to listen for any Clerk event.
+
+## Set up ngrok
+
+To test a webhook locally, you will need to expose your local server to the internet. For this guide, you will use [ngrok](https://ngrok.com/).
+
+1. Go to the [ngrok website](https://ngrok.com/) and create an account.
+2. Once you have made it to the [ngrok dashboard](https://dashboard.ngrok.com/get-started/setup/macos), in navigation sidebar, select [**Domains**](https://dashboard.ngrok.com/cloud-edge/domains).
+3. Select **Create domain**.
+4. ngrok will generate a free, non-ephemeral domain for you and a command to start a tunnel with that domain. The command should look something like this:
+
+`ngrok http --domain=fawn-two-nominally.ngrok-free.app 80`
+
+This command will start a tunnel from `https://fawn-one-nominally.ngrok-free.app` to your local server on port 80.
+5. Change the port number to whatever your server is running on. For this guide, change it to 3000 and then run the command in your terminal.
```

Comment:
Bullets 5 and 6 didn't render on new lines for me. Might be a one time issues, but just check they are new lines for you.

Reply:
Here's what I see in the deploy preview

<img width="751" alt="Screenshot 2024-04-30 at 12 15 51" src="https://github.com/clerk/clerk-docs/assets/98043211/85bf67c5-d927-4ff2-be9b-cc964cd9585a">